src/components/osc-unique/oscUnique.js

```diff
@@ -31,21 +31,41 @@ angular.module('openshiftCommonUI')
     return {
       restrict: 'A',
       scope: {
-        oscUnique: '='
+        oscUnique: '=',
+        oscUniqueDisabled: '='
       },
       require: 'ngModel',
       link: function($scope, $elem, $attrs, ctrl) {
         var list = [];
+        var disabled = false;
+        var isValid = true;
```

Comment:
I suggest `isUnique` here. It's technically valid if disabled even when not unique, so I think `isUnique` is more accurate.